packages/core/src/Room.ts

```diff
@@ -336,6 +422,9 @@ export abstract class Room<State= any, Metadata= any> {
     }
   }
 
+  /**
+   * This method will check whether mutations have occurred in the state, and broadcast them to all connected clients.
```

Comment:
The editor already tells the user this is a method, so in cases like this we can remove `"This method will"` (or equally redundant words to describe a scope already known by the editor)

I suggest:
> "Checks whether mutations have occurred in the state, and broadcast them to all connected clients."